app/gzac/src/main/java/com/ritense/gzac/listener/ApplicationReadyEventListener.kt

```diff
@@ -116,164 +116,131 @@ class ApplicationReadyEventListener(
             createBomenObjectManagement(objecttypenApiPluginId, objectenApiPluginId)
             createVerzoekPlugin(notificatiesApiPluginId, bezwaarConfigurationId)
             createSmartDocumentsPlugin()
-            val protaalTaakPluginId = createPortaalTaakPlugin(notificatiesApiPluginId, taakConfigurationId)
-            createPortaalTaakLink(protaalTaakPluginId)
-            completePortaalTaakLink(protaalTaakPluginId)
-            portalTaskLinkDocumentToZaak(zakenApiPluginId)
+            val protaalTaakPluginId = createPortaaltaakPlugin(notificatiesApiPluginId, taakConfigurationId)
```

Comment:
Would you mind changing this to portaalTaakPluginId? And then its usages of course.